packages/slice-machine/server/src/api/index.ts

```diff
@@ -298,6 +300,24 @@ router.post(
   })
 );
 
+// Sentry Proxy
+// The Sentry client send a POST request with a plaintext body
+// Not supported by express by default
+router.use(function (req, _ /* res */, next) {
+  if (req.is("text/*")) {
+    req.body = "";
+    req.setEncoding("utf8");
+    req.on("data", function (chunk) {
+      req.body += chunk;
+    });
+    req.on("end", next);
+  } else {
+    next();
+  }
+});
+// eslint-disable-next-line @typescript-eslint/no-misused-promises,
+router.post("/sentry", WithEnv(sentryHandler));
```

Comment:
probably best not to call the route sentry as we've had issues with using obvious route names before ;)

Reply:
Any suggestion? :-)

Reply:
Updated